ui/perfherder/alerts/AlertsView.jsx

```diff
@@ -260,6 +275,7 @@ class AlertsView extends React.Component {
             </Alert>
           )}
           <AlertsViewControls
+            frameworks={alertsViewFrameworks}
```

Comment:
We don't need to pass it further down. `AlertsViewControls` doesn't use it.

Reply:
`AlertsViewControls` does not uses it, but it already receives `frameworks` via `{...this.props}`.
I think that it is better to maintain consistence of data in our `AlertsView` sub-app.

Reply:
I see. Could we at least move this down, right above `{...this.props}`?

Reply:
I don't understand why this is being added here. Since `frameworks` is indeed being passed in via `this.props`, this can cause confusion as to which value is being used if that prop is in fact used in `AlertsViewControls` in the future. I'm also not a fan of renaming props as they're passed down to other components because it makes it harder to trace through the app.

Reply:
Do you want to remove the `frameworks={alertsViewFrameworks}` line and let the `frameworks` to be passed via `{...this.props}`, right?

Reply:
Yes, it will save someone a headache down the road.

Reply:
Done.